buildSrc/src/main/kotlin/Osgi.kt

```diff
@@ -83,6 +83,18 @@ fun Project.applyOsgiMultiplatform(vararg bndProperties: String) {
 
       override fun getClassesTaskName() = "${jvmMainSourceSet.classesTaskName}ForFakeMain"
 
+      override fun getCompileOnlyConfigurationName(): String = jvmMainSourceSet.compileOnlyConfigurationName + "ForFakeMain"
```

Comment:
@swankjesse is this safe? or rather are our OSGi tests good enough to catch this?

I'll probably land, assuming the tests would fail if not.

```
> Task :okhttp-osgi-tests:test
```